test/useWith.js

```diff
@@ -8,12 +8,12 @@ describe('useWith', function() {
   function add1(x) { return x + 1; }
   function mult2(x) { return x * 2; }
   function div3(x) { return x / 3; }
-  var f = R.useWith(max, add1, mult2, div3);
+  var f = R.useWith(max, [add1, mult2, div3]);
 
   it('takes a arbitrary number of function arguments and returns a function', function() {
```

Comment:
This description is no longer accurate. Let's state that it _takes a function and a list of functions and returns a function_.